src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/conversion.cpp

```diff
@@ -145,6 +145,11 @@ void ConvertCPULayerTest::SetUp() {
     auto conversion = std::make_shared<ov::op::v0::Convert>(params.front(), outPrc);
 
     function = makeNgraphFunction(inPrc, params, conversion, "ConversionCPU");
+
+    if (special_value == ov::test::SpecialValue::none && outPrc == ov::element::f8e4m3) {
```

Comment:
It should be inference precision dependant, not parameter dependant

Reply:
based on offline discussion, have created a ticket to follow up this issue.